src/Language/Printer.php

```diff
@@ -181,12 +182,12 @@ public function printAST($ast)
                             . $node->selectionSet;
                     },
 
-                    NodeKind::INT => static function (IntValueNode $node) {
-                        return $node->value;
+                    NodeKind::INT => static function (IntValueNode $node) : string {
+                        return strval($node->value);
```

Comment:
```suggestion
                        return (string) $node->value;
```

Faster, since it is not a function call. As far as i know, it does the same. I think we should replace it everywhere.

Reply:
Fair enough. 466f26b519f5b48ae2d524c5653a98a6cbb0f420